### ISSUE TITLE
migrate away from @diafygi's acme-tiny

### ISSUE THREAD

**Issue by James-E-A** [JamesTheAwesomeDude/encryptme#2]
It would be nice to have the program actually have an API into Let's Encrypt instead of just grokking it with an (admittedly very nice) script

**Comment by James-E-A** [comment]
This would free us from [all these](https://github.com/diafygi/acme-tiny/issues), too

**Comment by James-E-A** [comment]
Needs to happen after #1 unless we want to tango with curl

**Comment by James-E-A** [comment]
Orrrr
https://github.com/google/acme

Managed by Don't Be Evil so you know it'll probably be at least "robust" although the dependency on Go sucks so perhaps not

>but dat simple syntax tho
